hw/ip/pwrmgr/data/pwrmgr.hjson

```diff
@@ -28,7 +28,7 @@
     },
 
     { struct:  "pwr_clk",
-      type:    "req_rsp",
+      type:    "broadcast",
```

Comment:
I have plan to revise the intermodule signal format. But don't worry here. I will do it at once!

Reply:
thanks!

Reply:
Could you change this to `"uni"`?

Reply:
thanks for catching that.   Did this not fail validate because there's no inter_signal declared in top_earlgrey?